File: TransCoder_tokenized_test_set_functions/cpp/FIND_THE_ELEMENT_THAT_APPEARS_ONCE_2.cpp
int singleNumber ( int a [ ], int n ) {
  unordered_set < int > s ( a, a + n );
  int arr_sum = accumulate ( a, a + n, 0 );
  int set_sum = accumulate ( s . begin ( ), s . end ( ), 0 );
  return ( 3 * set_sum - arr_sum ) / 2;
}

File: TransCoder_tokenized_test_set_functions/java/FIND_THE_ELEMENT_THAT_APPEARS_ONCE_2.java
static int singleNumber ( int a [ ], int n ) {
  HashSet < Integer > s = new HashSet < Integer > ( );
  for ( int i : a ) {
    s . add ( i );
  }
  int arr_sum = 0;
  for ( int i : a ) {
    arr_sum += i;
  }
  int set_sum = 0;
  for ( int i : s ) {
    set_sum += i;
  }
  return ( 3 * set_sum - arr_sum ) / 2;
}